cpp/ql/src/semmle/code/cpp/Type.qll

```diff
@@ -326,6 +352,10 @@ private predicate isArithmeticType(@builtintype type, int kind) {
 
 /**
  * The C/C++ arithmetic types. See 4.1.1.
+ *
+ * This includes primitive types on which arithmetic, bitwise or logical
+ * operations may be performed.  Examples of arithmetic types include
+ * `char`, `int`, `float` and `bool`.
```

Comment:
```suggestion
 * `char`, `int`, `float`, and `bool`.
```

Reply:
I think either is correct, but I'll admit to having the same preference.